aten/src/ATen/NestedTensorImpl.h

```diff
@@ -92,10 +96,15 @@ struct TORCH_API NestedTensorImpl : public c10::TensorImpl {
    */
   at::Tensor get_unsafe_storage_as_tensor() const {
     auto buffer_key_set_ = generate_buffer_key_set();
-    const auto buffer_size = get_buffer_size();
     auto buffer_tensor_impl = c10::make_intrusive<TensorImpl>(
         c10::TensorImpl::VIEW, Storage(storage_), buffer_key_set_, data_type_);
-    buffer_tensor_impl->set_sizes_contiguous(c10::makeArrayRef(buffer_size));
+    if (C10_UNLIKELY(this->is_meta())) {
+      buffer_tensor_impl->set_sizes_and_strides(
+          {c10::SymInt(0)}, {c10::SymInt(1)}, c10::nullopt);
```

Comment:
Q: is this okay or should we be propagating the same unbacked SymInt used to construct the buffer?

Reply:
I have less context here but baking in the 0 and 1 symint doesn't look right (really helpful feedback lol)

Reply:
This doesn't look right to me either. How come get_buffer_size isn't returning a SymInt?